src/prefect/tasks/snowflake/snowflake.py

```diff
@@ -15,9 +17,9 @@ class SnowflakeQuery(Task):
              package documentation for details
         - user (str): user name used to authenticate
         - password (str, optional): password used to authenticate.
```

Comment:
As a general comment on docstrings, I'd love for us to move to correct sentence structure for argument descriptions: Capitalized first word unless we're referencing a literal. Period at end of sentence. In this case it's consistent as is and I'm okay with leaving it that way until further work is needed. Your call.

Reply:
To clarify, both styles are still reflected in task docstrings. I would like to move v1 docs to sentence style as it makes sense to do so. I am enforcing sentence style in v2.